test/fixtures/scenarioManager.js

```diff
@@ -0,0 +1,81 @@
+var path = require('path');
+var config = require('./config');
+var elasticsearch = require('elasticsearch');
+
+function ScenarioManager(server) {
+  if (!server) throw new Error('No server defined');
+
+  this.client = new elasticsearch.Client({
+    host: server
+  });
+}
+
+/**
+* Load a testing scenario
+* @param {string} id The scenario id to load
+* @return {Promise} A promise that is resolved when elasticsearch has a response
+*/
+ScenarioManager.prototype.load = function (id) {
+  var scenario = config[id];
+  if (!scenario) throw new Error('No scenario found for ' + id);
+
+  var self = this;
+  return Promise.all(scenario.bulk.map(function bulk(file) {
+    var mapping;
+
+    if (scenario.mapping) {
```

Comment:
Again here, this is more than a mapping, maybe call it indexDefinition or something?